examples/amp-story/amp-story-shopping.html

```diff
@@ -45,7 +45,7 @@
           <amp-story-shopping-tag data-tag-id="sunglasses" ></amp-story-shopping-tag>
           <amp-story-shopping-tag data-tag-id="backpack" ></amp-story-shopping-tag>
         </amp-story-grid-layer>
-        <amp-story-shopping-attachment layout="nodisplay"></amp-story-shopping-attachment>
+        <amp-story-shopping-attachment layout="fill"></amp-story-shopping-attachment>
```

Comment:
Why `fill` over `contain`? Do you need to specify a layout at all?

Reply:
TIL: layout does not need to be defined if `layout == Layout_Enum.CONTAINER` :)